.env.example

```
@@ -247,6 +247,13 @@ SHU_API_KEY=""
 SHU_API_KEY_USER_EMAIL=""
 SHU_SECRET_KEY=""               # pragma: allowlist secret
 
+# Password policy: moderate (default) or strict
+# moderate: min 8 chars, uppercase, lowercase, number
+# strict: all moderate rules + at least one special character
+SHU_PASSWORD_POLICY="moderate"               # pragma: allowlist secret
+SHU_PASSWORD_MIN_LENGTH=8
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SHU_PASSWORD_MIN_LENGTH key should go before the SHU_SECRET_KEY key

(UnorderedKey)